sample/kotlin/src/main/kotlin/com/datadog/android/sample/home/HomeFragment.kt

```diff
@@ -15,7 +15,7 @@ import androidx.core.view.children
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProviders
 import androidx.navigation.NavController
-import androidx.navigation.fragment.NavHostFragment.findNavController
+import androidx.navigation.fragment.NavHostFragment.Companion.findNavController
```

Comment:
Why do we need to specify `Companion` now? Is it related to the Kotlin 1.8 change?